SPECIFICATION/README.md

```diff
@@ -0,0 +1,185 @@
+//
+// => Signed Commit Messages document TRANSACTIONS and/or RULE CHANGES
+// => All "network people" have an up-to-date fork of the "data" git repo
+// @TODO: github accounts and peoples trust assure uniqueness of
+// @TODO: Assets Journal => EUR or USD bank account & asset ownership contracts
+// Event:
+pull -> read from files -> apply function -> write to file system -> push
+// Event Queue:
+if 2 events happen in same time, second event gets enqueued
+//
+```
+
+# Micro Proposals
+
+#### Rule Ideas **`@serapath`**
+1. `People` can `join` (=merge) and `leave` (=fork) the `network` voluntarily
+* `People` vote `median` for `Revenue Tax` which fills `Assets Journal`
+* `People` vote `median` for `Reputation Price` to be paid to others for them to **respect reputation**
+* `People` vote `median` for `Creation Multiplier` to change resolution of current `Reputation Points` and `Issue Prices`
+* `People` vote and `median` is `Tick Duration` which drives `Engine`
+  * e.g. could initially be set to one day per journal update
+  * e.g. journal updates get scheduled until the tick ends and executes all
+* `External Income` is distributed to `people` based on the `Network Shares`
+* `External Expenses` are paid from the `Assets Journal`
+* `People` can post/offer wishes/products/services/activities to the `Issue Board`
+* `People` can make **PULL REQUESTS** for change proposals to the `Network`
+  * **`REASONS:`**
+    > 1. Every update to the RepPointsServer should be made as a Pull Request and not directly pushed to this repository, so we can discuss and agree on the update before we merge it.
+
+* `People` can `prioritize` *(important/urgent)* items on the `Taskboard`
+* `People` vote on `median` is `Holding Tax` which fills `Assets Journal`
+* Evolving set of `Standard Contracts`
+  Every `standard contract` can be a one time thing, or a re-occuring thing.
+  1. `pledge` => for example: i give 10 points if everyone else gives 10 points too
+  2. `kickstarter` => Everyone can give points and something gets done when A points are reached
+  3. `wish` => I give B points to everyone that does a certain task i propose
+    * maybe I can add that after the task is finished I am allowed to give B+/-reward instead of B
+* People who join can never be kicked out - If somebody is not happy he should create his *organizational fork* and may invite others to join and continue on his/her/their own with the new organization.
+
+#### Rule Ideas **`@nasdneb`**
+* There should be a **"Voting System based on Reputation Points"**
+* Everyone starts with 10 points
+* Redistribution: Redistribute 5% each day  
+  * `Reasoning`: To have fast impact that helps to get a feeling for the mechanics, thus "how it works")  
+
+* Creation: Create 10 new points each week
+* It should be possible to transfer points to a user
+* Everyone makes his list of activities, and we can distribute our points. (i guess the 10 points)
+
```

Comment:
**Regarding Points:**
* there could be initialle 1 point and every founder or person who is in the network when it starts using the system gets a share of that 1 points, which could be A:0.1, B:0.3, C:0.2, D:0.1, E:0.3 in case it's 5 founders and they agree that's how much they invested until now.
From then on, a simple multiplier which would affect all TASK PRICES and CURRENT ACCOUNT BALANCES, e.g. a multiplier of 100 would increase all prices of tasks currently on the task board or that have already be done by 100 and the account balances would change to A:10, B:30, C:20, D:10, E:30